src/providers/boleto-api-bradesco-shopfacil/formatter.js

```diff
@@ -0,0 +1,53 @@
+const moment = require('moment')
+const {
+  path,
+  pipe,
+  toLower,
+  toUpper,
+  trim,
+} = require('ramda')
+
+const brazilianStates = require('../../lib/helpers/brazilian-states')
+
+const formatDate = timestamp => moment(timestamp).format('YYYY-MM-DD')
+
+const getDocumentType = (documentNumber) => {
+  const documentLength = String(documentNumber).trim().length
+
+  if (documentLength === 11) {
+    return 'CPF'
+  }
+
+  return 'CNPJ'
+}
+
+const formatStateCode = (boleto, from) => {
+  const possibilities = ['payer_address', 'company_address']
+
+  if (!possibilities.includes(from)) {
+    return ''
+  }
+
+  return pipe(
+    path([from, 'state']),
+    toLower,
+    trim,
+    state => brazilianStates[state] || state,
+    toUpper
+  )(boleto)
+}
+
+const getPayloadWithoutAuth = (payload) => {
+  const payloadWithoutAuth = Object.assign({}, payload)
+
+  delete payloadWithoutAuth.authentication
+
+  return payloadWithoutAuth
+}
```

Comment:
```suggestion
const getPayloadWithoutAuth = dissoc('authentication')
```

Ref: https://ramdajs.com/docs/#dissoc

Reply:
Mudado, Senhor Ramda!